lexicons/app/bsky/feed/defs.json

```diff
@@ -186,6 +194,10 @@
         "repost": { "type": "string", "format": "at-uri" }
       }
     },
+    "skeletonReasonPin": {
```

Comment:
I actually like that this could be used in the future for a feed gen to "pin" a post to the top of a feed